src/o365/graph/commands/schemaextension/schemaextension-list.ts

```diff
@@ -0,0 +1,155 @@
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Get a list of schemaExtension objects created in the current tenant, that can be InDevelopment, Available, or Deprecated.';
+  }
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.status = typeof args.options.status !== 'undefined';
+    telemetryProps.owner = typeof args.options.owner !== 'undefined';
+    telemetryProps.pageNumber = typeof args.options.pageNumber !== 'undefined';
+    telemetryProps.pageSize = typeof args.options.pageSize !== 'undefined';
+    return telemetryProps;
+  }
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const filter: string = this.getFilter(args.options);
+    let url = `${this.resource}/v1.0/schemaExtensions?$select=*${(filter.length > 0 ? '&' + filter : '')}`;
+    ((): Promise<any> => {
+      if (args.options.pageNumber && Number(args.options.pageNumber) > 0) {
+        const rowLimit: string = `&$top=${Number(args.options.pageSize ? args.options.pageSize : 10) * Number(args.options.pageNumber)}`;
+        url += rowLimit;
+        const requestOptions: any = {
+          url: url,
+          headers: {
+            accept: 'application/json;odata.metadata=none',
+            'content-type': 'application/json'
+          },
+          json: true
+        };
+        return request.get(requestOptions);
+      } else {
+        return Promise.resolve();
+      }
+  })().then((res:any):void =>{
+    const requestOptions: any = {
+      url: args.options.pageNumber && Number(args.options.pageNumber) > 0 ?res[`@odata.nextLink`] :url,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json'
+      },
+      json: true
+    };
+    request.get(requestOptions)
+      .then((res: any): void => {
+        cmd.log(res);
+        if (this.verbose) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+
+        cb();
+      }, (err: any) => this.handleRejectedODataJsonPromise(err, cmd, cb));
+  });
+  }
+
+  private getFilter(options: any): string {
+    const filters: any = {};
+    const excludeOptions: string[] = [
+      'pageNumber',
+      'pageSize',
+      'debug',
+      'verbose',
+      'output'
+    ];
+
+    Object.keys(options).forEach(key => {
+      if (excludeOptions.indexOf(key) === -1) {
+        filters[key] = encodeURIComponent(options[key].replace(/'/g, `''`));
+      }
+    });
+    let filter: string = Object.keys(filters).map(key => `eq(${key}, '${filters[key]}')`).join(' and ');
+    if (filter.length > 0) {
+      filter = '$filter=' + filter;
+    }
+
+    return filter;
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-s, --status [status]',
```

Comment:
We should provide the autocomplete option for the status: `autocomplete: ['Available', 'InDevelopment', 'Deprecated']`